make/test.mk

```diff
@@ -26,3 +38,41 @@ test-with-coverage:
 upload-codecov-report: 
 	# Upload coverage to codecov.io
 	bash <(curl -s https://codecov.io/bash) -f $(COV_DIR)/coverage.txt -t 51cc45ad-2e54-4e68-98cb-8ab15cf2b8df
+
+############################################################
+#
+# End-to-end Tests
+#
+############################################################
+
+.PHONY: test-e2e
```

Comment:
Ideally this goal will be run with an extra job on OpenShift CI. I'll see if it's doable